bioio_bioformats/reader.py

```diff
@@ -14,6 +15,7 @@
 from .biofile import BioFile
 
 ###############################################################################
+log = logging.getLogger(__name__)
```

Comment:
Does this actually get used anywhere?

Reply:
Not anymore; a remnant of when this ticket was going to be about logging.